src/test/kotlin/sudoklin/data/SudokuConstructionTest.kt

```diff
@@ -55,5 +57,22 @@ class NewSudokuTest : Spek({
                 }
             }
         }
+
+        it("should have one less number in the row/column/group if it was added") {
```

Comment:
can we re-phrase this?
"should have one less number in the row/column/group if a cell was solved"